src/components/table/expandable/cell/style.css

```diff
@@ -30,10 +29,10 @@
 :host[open] smoothly-icon {
 	transform: rotate(90deg);
 }
-aside {
+:host > div {
 	display: grid;
+	grid-template-columns: auto 1fr;
 	grid-auto-flow: column;
```

Comment:
using template with auto 1fr makes this rule unnecessary i think?

Reply:
Without it the expandable cell/row arrow takes up half the space of the cell.